integration_tests/src/main/python/date_time_test.py

```diff
@@ -398,12 +404,13 @@ def invalid_date_string_df(spark):
     return spark.createDataFrame([['invalid_date_string']], "a string")
 
 @pytest.mark.parametrize('ansi_enabled', [True, False], ids=['ANSI_ON', 'ANSI_OFF'])
+@pytest.mark.parametrize('non_utc_timezone_enabled', [True, False], ids=['NOT_UTC_ON', 'NOT_UTC_OFF'])
 @pytest.mark.parametrize('data_gen,date_form', str_date_and_format_gen, ids=idfn)
-@allow_non_gpu(*non_utc_allow)
-def test_string_to_unix_timestamp(data_gen, date_form, ansi_enabled):
+@allow_non_gpu('ProjectExec')
```

Comment:
I don't want to have a hard coded ProjectExec that does not know if we should fall back to the CPU or not!.